apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -65,6 +65,9 @@ export const BreakpointsSelector = ({
         type="single"
         value={selectedBreakpoint.id}
         onValueChange={(breakpointId: string) => {
+          if (!breakpoints.has(breakpointId)) {
```

Comment:
we are not writing stuff like `!something` because when it comes to readability you may get `!i` which is super hard to read on many fonts